### PR TITLE
fix: test failure on firefox

### PR DESCRIPTION
**Pull request by pedronfigueiredo**
## **Description**

Fixes a pipeline error on firefox (only visible on develop).

https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/91929/workflows/4e066691-a4c8-43f9-bd9d-c90335d82613/jobs/3420611/tests#failed-test-0

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25895?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -208,7 +208,7 @@ workflows:
             - prep-build-test-mv2
             - get-changed-files-with-git-diff
       - test-e2e-firefox-confirmation-redesign:
-          <<: *develop_master_rc_only
+          # <<: *develop_master_rc_only
```
Comment (danjm) [inline .circleci/config.yml:211]: Is this change intended to be part of this PR?

Comment (pedronfigueiredo) [reply .circleci/config.yml:211]: This is just to test the fix addresses the problem on CI, as per [this conversation](https://consensys.slack.com/archives/CTQAGKY5V/p1721222766599959?thread_ts=1721211621.814389&cid=CTQAGKY5V).

I added the `DO-NOT-MERGE` label to the PR and will remove the change before merging.

**Review by jpuri — APPROVED** [review]
Changes look good, just curious why this `await toggleAdvancedDetails(driver);` makes it flaky for FF.

**Review by pnarayanaswamy — APPROVED** [review]

**Review by seaona — APPROVED** [review]
thank you so much for the quick fix @pedronfigueiredo :raised_hands:

**Review by matthewwalsh0 — APPROVED** [review]
